documentation/api/io.strimzi.api.kafka.model.CruiseControlSpec.adoc

```diff
@@ -49,9 +49,12 @@ spec:
   cruiseControl:
     # ...
     config:
+      # Note that `default.goals` must be a superset of `hard.goals`
```

Comment:
Looks good. I might only make the general suggestion for all the notes:

> Note that `default.goals` (superset) must also include all `hard.goals` (subset)

Reply:
Updated the comments to match your note